packages/block-library/src/common.scss

```diff
@@ -120,3 +120,13 @@ html :where(.has-border-color) {
 html :where([style*="border-width"]) {
 	border-style: solid;
 }
+
+
+/**
+ * Provide baseline responsiveness for images.
+ */
+
```

Comment:
```suggestion
```

(It looks like we're inconsistent about this, but the code comment above has no space between the comment and the code. 🤷)